todo-app/src/App.js

```diff
@@ -1,10 +1,68 @@
 import React, { Component } from 'react';
+import "./style.css"
+import Header from "./components/Header.js"
+import TodoList from "./components/TodoList"
+import myTodoList from "./components/myTodoList.json"
+import moment from "moment"
+import AddingContainer from "./components/AddingContainer"
+
 
 class App extends Component {
+  state = {
+    tasks: myTodoList
+  }
+  componentWillUpdate(nextProps, nextState) {
+    localStorage.setItem("todo list", JSON.stringify(nextState.tasks))
+  }
+  componentDidMount() {
+    if (localStorage.getItem("todo list")) {
+      this.setState({ tasks: JSON.parse(localStorage.getItem("todo list")) })
+    }
+  }
+  nextId = () => {
```

Comment:
As discussed in class, this is not a safe way to generate IDs. Please make sure to change this for next week!